src/test/python/apache/aurora/config/test_loader.py

```diff
@@ -93,6 +101,27 @@ def test_load_json_single():
   assert new_job == job
 
 
+def test_load_json_memoized():
```

Comment:
Is this actually testing memoization? Shouldn't it be asserting that the pystachio evaluation/loading isn't done on the second request? I think this test would pass if you pass in is_memoized=False as well.

Reply:
1) write the json with job[0] and read it with memoizing and assert that I have job[0]
2) overwrite the config with write the json for job[1]  and read with memozation and assert I still have job[0] (cached content)
3) read without memoization and assert I have job[1]

Reply:
I commented.